tsdef/types.d.ts

```diff
@@ -176,6 +201,10 @@ interface IPropertyPath {
   contents: [BindingForm, string, string];
 }
 
+interface ITypePropertyPath {
+  tag: "TypePropertyPath";
+}
```

Comment:
@wodeni Should this type be just a tag, with no contents? I wasn't sure what this corresponded to in the Haskell grammar.

Reply:
Oh it’s probably left in the HS file. We are not actively using this, right? I thought we opted for the [] access instead

Reply:
I see, this is the type property access, like `v.x`. Yeah, we aren't actively using it. Thanks for clarification.